.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,5 +1,10 @@
 ---
 name: Bug Report
+about: Issues, deficiencies or problems discovered on Solidity.
+title: ''
+labels: 'bug :bug:'
```

Comment:
Not sure there is a point in adding it back. We removed labels from these templates because they don't work. They only get applied when we create issues ourselves. Users don't have permissions to add labels.

Reply:
What about this? I'm not kidding. It works that way. We'll end up having these labels only on issues we create ourselves.